apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisDataTable.vue

```diff
@@ -270,6 +275,11 @@ const handleExcelDownload = async () => {
 const handleExport = async () => {
     await handleExcelDownload();
 };
+const handleUpdateThisPage = async () => {
+    const { results, more } = await listCostAnalysisTableData();
+    tableState.items = getRefinedChartTableData<CostAnalyzeRawData>(results, costAnalysisPageState.granularity, costAnalysisPageState.period ?? {});
```

Comment:
issue: ahh... This PR will be conflict.. I modified `getRefinedChartTableData` function.

Reply:
Oh I'll delete generic!